src/palette/internal/palettecreator.cpp

```diff
@@ -1547,65 +1547,38 @@ PalettePtr PaletteCreator::newTextPalette(bool defaultPalette)
-    //: For brass and plucked string instruments: staff text that prescribes to use mute while playing, see https://en.wikipedia.org/wiki/Mute_(music)
-    sp->appendElement(mu, QT_TRANSLATE_NOOP("palette", "Mute"))->setElementTranslated(true);
-
-    auto no = makeElement<PlayTechAnnotation>(gpaletteScore);
-    //: For brass and plucked string instruments: staff text that prescribes to play without mute, see https://en.wikipedia.org/wiki/Mute_(music)
-    no->setXmlText(QT_TRANSLATE_NOOP("palette", "open"));
-    no->setTechniqueType(PlayingTechniqueType::Open);
-    //: For brass and plucked string instruments: staff text that prescribes to play without mute, see https://en.wikipedia.org/wiki/Mute_(music)
-    sp->appendElement(no, QT_TRANSLATE_NOOP("palette", "Open"))->setElementTranslated(true);
-
-    auto distort = makeElement<PlayTechAnnotation>(gpaletteScore);
-    distort->setXmlText(QT_TRANSLATE_NOOP("palette", "distort"));
-    distort->setTechniqueType(PlayingTechniqueType::Distortion);
-    sp->appendElement(distort, QT_TRANSLATE_NOOP("palette", "Distortion"))->setElementTranslated(true);
-
-    auto overdrive = makeElement<PlayTechAnnotation>(gpaletteScore);
-    overdrive->setXmlText(QT_TRANSLATE_NOOP("palette", "overdrive"));
-    overdrive->setTechniqueType(PlayingTechniqueType::Overdrive);
-    sp->appendElement(overdrive, QT_TRANSLATE_NOOP("palette", "Overdrive"))->setElementTranslated(true);
-
-    auto harmonics = makeElement<PlayTechAnnotation>(gpaletteScore);
-    harmonics->setXmlText(QT_TRANSLATE_NOOP("palette", "harmonics"));
-    harmonics->setTechniqueType(PlayingTechniqueType::Harmonics);
-    sp->appendElement(harmonics, QT_TRANSLATE_NOOP("palette", "Harmonics"))->setElementTranslated(true);
-
-    auto jazzTone = makeElement<PlayTechAnnotation>(gpaletteScore);
-    jazzTone->setXmlText(QT_TRANSLATE_NOOP("palette", "jazz tone"));
-    jazzTone->setTechniqueType(PlayingTechniqueType::JazzTone);
-    sp->appendElement(jazzTone, QT_TRANSLATE_NOOP("palette", "Jazz tone"))->setElementTranslated(true);
-
-    auto normal = makeElement<PlayTechAnnotation>(gpaletteScore);
-    normal->setXmlText(QT_TRANSLATE_NOOP("palette", "normal"));
-    normal->setTechniqueType(PlayingTechniqueType::Natural);
-    sp->appendElement(normal, QT_TRANSLATE_NOOP("palette", "Normal"))->setElementTranslated(true);
+    struct PlayTechAnnotationInfo {
+        const char* xmlText;
+        PlayingTechniqueType playTechType;
+        bool expressionType = false;
+        muse::TranslatableString userName = TConv::userName(playTechType);
+    };
+
+    static const std::vector<PlayTechAnnotationInfo> playTechAnnotations = {
+        { QT_TRANSLATE_NOOP("palette", "legato"),    PlayingTechniqueType::Legato },
+        { QT_TRANSLATE_NOOP("palette", "pizz."),     PlayingTechniqueType::Pizzicato },
+        { QT_TRANSLATE_NOOP("palette", "arco"),      PlayingTechniqueType::Natural, false, muse::TranslatableString("palette", "arco") },
```

Comment:
Micro remark: Shouldn't the second one be uppercase for consistency?
```suggestion
        { QT_TRANSLATE_NOOP("palette", "arco"),      PlayingTechniqueType::Natural, false, muse::TranslatableString("palette", "Arco") },
```

Reply:
Ah yes, has been fixed!